backend/infrastructures/apps/load/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+# Create your models here.
+class OutputData(models.Model):
+    full_name = models.CharField(max_length=255)
```

Comment:
As mentioned, we will use JSONB field to store data.

To regenerate migrations use:
manage.py migrate load --zero
rm load/migrations/0001.py
manage.py makemigrations